### DISCUSSION TITLE
Improved resource collecting

### DISCUSSION THREAD

**Discussion by penfriendz** [TheSuperHackers/GeneralsGamePatch#2553] [Ideas]
i manually send gatherers to different supply piles. this results in a noticeable increase in income.
while it is good for me to do this and have others not know about it, it's also a little fiddly, especially to know which gatherer is going to which pile. also, after the pile is gone, all the workers stack back on the same pile again which is a bit annoying

**Comment by xezon** [comment]
Should be an issue report with enhancement label.

Perhaps a useful improvement here would be have workers continue collecting at the nearest supply pile that is closest to the last exhausted pile, instead of the current worker position.

And if there was no exhausted pile, then look for closest resource around worker position as usual.

Also: There is this proposal:
* #624
